boards/acd52832/include/gpio_params.h

```diff
@@ -33,19 +34,25 @@ extern "C" {
 static const  saul_gpio_params_t saul_gpio_params[] =
 {
     {
-        .name = "LED 1",
         .pin = LED0_PIN,
         .mode = GPIO_OUT,
         .flags = (SAUL_GPIO_INVERTED | SAUL_GPIO_INIT_CLEAR),
     },
-        {
-        .name = "Button 1",
+    {
         .pin  = BTN0_PIN,
         .mode = BTN0_MODE,
         .flags = SAUL_GPIO_INVERTED,
     }
 };
 
+/**
+ * @brief GPIO information for SAUL registry
+ */
+static const saul_reg_info_t saul_gpio_info[] =
+{
+    { .name = "LED 1" },
```

Comment:
waitaminute ... this adds a dependency from saul without registry to saul with registry...

Reply:
If saul_gpio module is being used (needed to define `saul_gpio_params_t`), then the auto init will register the gpios on the registry anyways, so I think it is needed.